.github/workflows/ci.yml

```diff
@@ -30,13 +30,13 @@ jobs:
   qa:
     strategy:
       matrix:
```

Comment:
```suggestion
      fail-fast: false
      matrix:
```

Reply:
Completely forgot we added this on the others :)